session/driver/file.go

```diff
@@ -80,7 +79,7 @@ func (f *File) Read(id string) (string, error) {
 		}
 	}
 
-	return "", errors.SessionNotFound.Args(id)
+	return "", nil
```

Comment:
Q: Does `errors.SessionNotFound` need to be deleted?

Reply:
Good question, we can remove it.